cartography/models/github/teams.py

```diff
@@ -0,0 +1,112 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class GitHubTeamNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('url')
```

Comment:
nonblocker: I feel like id should maybe be `{org}/{name}`, but i see that url was also used for the id of Repos